.editorconfig

```diff
@@ -0,0 +1,19 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+# Unix-style newlines with a newline ending every file
+end_of_line = lf
+insert_final_newline = true
+
+# Set default charset
+charset = utf-8
+
+# 2 space indentation
+tab_size = 2
+tab_style = space
+
+# trim any trailing whitespace
+trim_trailing_whitespace = true
```

Comment:
Is there anyway to whitelist specific files for this? For example, I think this has bitten me on `.md` files in the past?